src/lib/viewers/controls/annotations/AnnotationsButton.tsx

```diff
@@ -20,22 +24,46 @@ export default function AnnotationsButton({
     isEnabled = true,
     mode,
     onClick = noop,
+    setWasClosedByUser = noop,
+    tooltipFlowAnnotationsExperience = {
```

Comment:
Can we check for this value being defined and not render the tooltip if it's missing?

Reply:
I've updated the shouldTarget prop for `TargetedClickThroughGuideTooltip` to
```
const { tooltipFlowAnnotationsExperience } = experiences;
shouldTarget={showDrawing && tooltipFlowAnnotationsExperience && tooltipFlowAnnotationsExperience.canShow}
```
if shouldTarget equals false it will not render the tooltip. Did you want me to be more explicit? such as:
```
if (!tooltipFlowAnnotationsExperience) {
    return children;
}

return (
    <tooltipFlowAnnotationsExperience>
        {children}
    </tooltipFlowAnnotationsExperience>
);
```